content/en/docs/tasks/traffic-management/mirroring/index.md

```diff
@@ -136,10 +132,9 @@ you will apply a rule to mirror a portion of traffic to `v2`.
 
 ## Creating a default routing policy
 
-By default Kubernetes load balances across both versions of the `httpbin` service.
-In this step, you will change that behavior so that all traffic goes to `v1`.
+By default Kubernetes load balances across both versions of the `httpbin` service. In this step, you will change that behavior so that all traffic goes to `v1`.
```

Comment:
```suggestion
By default Kubernetes load balances across both versions of the `httpbin` service.
In this step, you will change that behavior so that all traffic goes to `v1`.
```
Wrapping long lines can be beneficial, making future edits and reviews more convenient.